frontend/sass/evictionfree/styles.scss

```diff
@@ -18,41 +16,41 @@ $jf-alt-title-family: "Montserrat Bold", sans-serif;
 @import "../_a11y.scss";
 @import "../_safe-mode.scss";
 @import "../_modal.scss";
-@import "../norent/_modal-overrides.scss";
+@import "../evictionfree/_modal-overrides.scss";
```

Comment:
If you're sure about copying over everything, you should probably set these to `import "./_modal-overrides.scss"` rather than going up a directory and back into `evictionfree` (since this file is inside `evictionfree`.